src/components/v5/common/ActionSidebar/partials/ActionSidebarWidgetLoadingSkeleton/ActionSidebarWidgetLoadingSkeleton.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+const displayName = `v5.common.ActionSidebar.partials.ActionSidebarWidgetLoadingSkeleton`;
+
+const ActionSidebarWidgetLoadingSkeleton = () => {
+  return (
+    <div className="relative flex flex-col gap-4">
+      <div className="flex w-full items-center gap-4">
+        <div className="flex items-center justify-center">
+          <div className="h-[10px] w-[10px] rounded-full bg-gray-100" />
+        </div>
+        <div className="h-6 w-14 overflow-hidden rounded-full skeleton" />
+      </div>
+      <div className="ml-[1.625rem] overflow-hidden rounded-lg border border-gray-200">
+        <div className="flex w-full gap-2 border-b border-gray-200 bg-gray-50 p-[1.125rem] py-3">
+          <div className="h-4 w-4 overflow-hidden rounded-md skeleton" />
+          <div className="h-4 w-52 overflow-hidden rounded-md skeleton" />
```

Comment:
Here actually we should replace `rounded-md` with `rounded-full`

Reply:
Looking at the Figma it looks like this should actually just be `rounded` (4px border radius). Rounded full would be too much here I think.